Cql/CodeGeneration.NET/AssemblyCompiler.cs

```diff
@@ -71,9 +71,10 @@ public AssemblyCompiler(
 
         public IReadOnlyDictionary<string, AssemblyData> Compile(
             LibrarySet librarySet,
-            DefinitionDictionary<LambdaExpression>? definitions = null)
+            DefinitionDictionary<LambdaExpression> definitions)

```

Comment:
This will require the user to create an empty dictionary themselves, why is that necessary?

Reply:
Resolved, since the usages to this method create them anyway